code/game/objects/items/weapons/grenades/grenade.dm

```diff
@@ -83,6 +85,16 @@
 
 /obj/item/grenade/proc/prime()
 
+/obj/item/grenade/proc/can_prime()
+	return TRUE
+
+/obj/item/grenade/proc/can_activate()
+	return TRUE
```

Comment:
Since "Activating" and "Priming" are basically the same thing, and I feel like these have similar functions, so it's probably unnecessary to have both. 

I realize Farie suggested `can_prime()` so CC: @farie82